bus_analyzer/bus_analyzer/main.py

```diff
@@ -218,6 +218,42 @@ def analyze_hour(db_manager, prototypes, existing_segments, base_dir, channel, q
 	for worker in workers:
 		worker.get() # re-raise errors
 
+	return [segment[1] for segment in segments_to_do]
+
+def post_process(db_manager, segments):
```

Comment:
some quick and dirty monitoring:
```suggestion
@timed(normalize=lambda result, db_manager, segments: len(segments))
def post_process(db_manager, segments):
```